randovania/games/planets_zebeth/gui/ui_files/planets_zebeth_game_export_dialog.ui

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
screenshot

Reply:
From randovania :

![image](https://github.com/user-attachments/assets/ab19a58e-07d8-45e7-b295-d33b1b130454)

From QT :
![image](https://github.com/user-attachments/assets/04bc6f0f-77cf-43a4-8dfc-1c43c4925815)

Reply:
No not into here
Post them as the intial or description or a close comment.

Reviewing big chunk of xml is not feasible, and these threads are too easy to get lost in.

Reply:
Oh ok